### PR TITLE
RHDEVDOCS-5845: Post GA changes in the GitOps 1.14 content

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

gitops-1.14, gitops-docs-main

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5845

Link to docs preview:

- [Release Note updates](https://82209--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes-1-14.html#gitops-release-notes-1-14-0_gitops-release-notes) for [GITOPS-5010](https://issues.redhat.com/browse/GITOPS-5010) , [GITOPS-4358](https://issues.redhat.com/browse/GITOPS-4358)

- Updates in the [Aggregated cluster roles](https://82209--ocpdocs-pr.netlify.app/openshift-gitops/latest/declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.html#aggregated-cluster-roles_customizing-permissions-by-creating-aggregated-cluster-roles) section. The 1st bullet in the Important Note is updated.

- Updates in the [Create user-defined cluster roles and configure user-defined permissions](https://82209--ocpdocs-pr.netlify.app/openshift-gitops/latest/declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.html#create-configure-aggregated-user-defined-permissions_customizing-permissions-by-creating-aggregated-cluster-roles) section. A small change is made in the first paragraph.

- Fixing old links in the Additional resources section of the [Managing the application set resources in non-control plane namespaces](https://82209--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_application_sets/managing-app-sets-in-non-control-plane-namespaces.html)

- Fixing the incorrect backtick of the default value `<default_Argo_CD_Known_Hosts>` of the `initialSSHKnownHosts` property in the [Argo CD custom resource properties table](https://82209--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties.html)

- Fixing an incorrectly rendered attribute in the [Using Argo Rollouts for progressive deployment delivery](https://82209--ocpdocs-pr.netlify.app/openshift-gitops/latest/argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.html) section.

QE review:
- [x] QE has approved this change.

Change Management
Need an ACK from Eng: anjoseph@redhat.com @iam-veeramalla
Need an ACK from PM: @siamaksade  @wtam2018
Need an ACK from Product experience: @RickJWagner
Need an ACK from QE: @varshab1210, asaleh@redhat.com
Need an ACK from DPM: kavverma@redhat.com
Need an ACK from CS: @Preeticp

Peer review: @Srivaralakshmi

Additional information:

### CODE REVIEW

**Comment by iam-veeramalla** [comment]
LGTM

File: modules/gitops-release-notes-1-14-0.adoc [hunk 1]
```diff
@@ -81,6 +83,8 @@ The signature verification does not function with the templated project field in
 
 * Before this update, the {gitops-title} Operator included a race condition where the OpenShift Service CA created the TLS secret before the default TLS termination policy was set. As a result, the default policy was set as `Passthrough` instead of `Reencrypt`. This update fixes this race condition by verifying whether the secret was created by the OpenShift Service CA, ensuring that `Reencrypt` is used as the correct default policy. link:https://issues.redhat.com/browse/GITOPS-4947[GITOPS-4947]
 
+* Before this update, you could not login to the Argo CD web console UI after a ROSA cluster, that includes a {gitops-shortname} Operator and an Argo CD instance configured with Dex-based SSO, resumed from hibernation. The login screen would display an error indicating an invalid redirect URI in the Dex configuration. This update fixes the issue by ensuring that the correct Dex redirect URL is updated in the Argo CD configuration whenever the Argo CD server route is modified. link:https://issues.redhat.com/browse/GITOPS-4801[GITOPS-4358]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-14-0.adoc:86]:
```suggestion
* Before this update, you could not login to the Argo CD web console UI after a ROSA cluster, that includes a {gitops-shortname} Operator and an Argo CD instance configured with Dex-based SSO, resumed from hibernation. The login screen would display an error indicating an invalid redirect URI in the Dex configuration. This update fixes the issue by ensuring that the correct Dex redirect URL is updated in the Argo CD configuration whenever the Argo CD server route is modified. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 Thanks for the rework. I have provided some suggestions. PTAL at them and implement.

**Comment by RickJWagner** [comment]
PLM has reviewed and approves this PR.
Thanks for your work, @Dhruv-Soni11 and @Srivaralakshmi

File: declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ With {gitops-title} 1.14 and later, as a cluster administrator, you can use aggr
 
 [IMPORTANT]
 ====
-* You can create aggregated cluster roles only for the Argo CD Application Controller component of a cluster-scoped Argo CD instance.
+* The Aggregated cluster roles functionality is optional and disabled by default. You can create aggregated cluster roles only for the Argo CD Application Controller component of a cluster-scoped Argo CD instance.
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc:23]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc [hunk 3]
```diff
@@ -43,6 +43,8 @@ For more information, refer to the link:https://argocd-operator.readthedocs.io/e
 
 * With this update, you can implement an alternative sharding algorithm that utilizes consistent hashing with bounded loads. The algorithm ensures a uniform distribution of managed clusters across shards while minimizing the need for reshuffling during configuration changes, such as adjustments in the number of managed clusters or shards. link:https://issues.redhat.com/browse/GITOPS-3584[GITOPS-3584]
 
+* With this update, {gitops-title} provides support for the `sidecar` and `initcar` containers for the Server and Application Controller components. You can configure these fields by using the `spec.server.sidecarContainers` and `spec.controller.sidecarContainers` for the `sidecar` container and the `spec.server.initContainers` and `spec.controller.initContainers` for the `initcar` container. link:https://issues.redhat.com/browse/GITOPS-5010[GITOPS-5010]
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:46]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:86]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc [hunk 4]
```diff
@@ -105,4 +130,14 @@ data:
...
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* WHen you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secrets applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that correspondeds to the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, you can verify whether the project field of an application aligns with the project field of the cluster secret. As a result, if a cluster secret is designated for project-a, an application assigned to project-b will be unable to access that secret. To use applications across multiple projects for cluster secrets, you must deselect the project field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```
Comment (anandf) [inline modules/gitops-release-notes-1-14-0.adoc:141]: Typo - `WHen` should be When

Comment (anandf) [reply modules/gitops-release-notes-1-14-0.adoc:141]: `no longer be classified as a global secret`

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc [hunk 5]
```diff
@@ -105,4 +131,14 @@ data:
...
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secret applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that correspondeds to the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, you can verify whether the project field of an application aligns with the project field of the cluster secret. As a result, if a cluster secret is designated for project-a, an application assigned to project-b will be unable to access that secret. To use applications across multiple projects for cluster secrets, you must deselect the project field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```
Comment (anandf) [inline modules/gitops-release-notes-1-14-0.adoc:142]: typo: `correspondeds`

Comment (anandf) [reply modules/gitops-release-notes-1-14-0.adoc:142]: Suggesting to reword as follows: Starting from {gitops-title} v1.14, cluster secrets that are scoped to the application's project alone would be considered for finding a matching  repo URL.

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:142]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc [hunk 6]
```diff
@@ -105,4 +131,14 @@ data:
...
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secret applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that corresponds to the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, cluster secrets that are scoped to an application's project can be considered for matching a `repo URL`. As a result, if a cluster secret is designated for project-a, an application assigned to project-b will be unable to access that secret. To use applications across multiple projects for cluster secrets, you must deselect the project field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```
Comment (anandf) [inline modules/gitops-release-notes-1-14-0.adoc:142]: if put in quotes `repo URL` is not correct. it should either be `repoUrl` we can also mention as `repository URL ` without any quotes.

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:142]: Fixed it as 'repoURL'.

**Comment by anandf** [comment]
LGTM. Reviewed the note about the breaking change as well.

**Comment by wtam2018** [comment]
LGTM

File: modules/gitops-release-notes-1-14-0.adoc [hunk 7]
```diff
@@ -81,6 +105,8 @@ The signature verification does not function with the templated project field in
 
 * Before this update, the {gitops-title} Operator included a race condition where the OpenShift Service CA created the TLS secret before the default TLS termination policy was set. As a result, the default policy was set as `Passthrough` instead of `Reencrypt`. This update fixes this race condition by verifying whether the secret was created by the OpenShift Service CA, ensuring that `Reencrypt` is used as the correct default policy. link:https://issues.redhat.com/browse/GITOPS-4947[GITOPS-4947]
 
+* Before this update, you could not login to the Argo CD web console UI after a ROSA cluster, that includes a {gitops-shortname} Operator and an Argo CD instance configured with Dex-based SSO, resumed from hibernation. The login screen would display an error indicating an invalid redirect URI in the Dex configuration. This update fixes the issue by ensuring that the correct Dex redirect URL is updated in the Argo CD configuration whenever the Argo CD server route is modified. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-14-0.adoc:108]:
```suggestion
* Before this update, you could not log in to the Argo CD web console UI after a ROSA cluster, that includes a {gitops-shortname} Operator and an Argo CD instance configured with Dex-based SSO, resumed from hibernation. The login screen would display an error indicating an invalid redirect URI in the Dex configuration. This update fixes the issue by ensuring that the correct Dex redirect URL is updated in the Argo CD configuration whenever the Argo CD server route is modified. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```

File: modules/gitops-release-notes-1-14-0.adoc [hunk 8]
```diff
@@ -105,4 +131,14 @@ data:
...
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secrets applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that corresponds to the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets that are scoped to an application's project will be considered for matching a repository URL. As a result, if a cluster secret is designated for example to project A, an application assigned to project B will be unable to access that secret. To enable the use of applications across various projects for cluster secrets, you must deselect the project field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-14-0.adoc:142]:
```suggestion
* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secrets applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL. As a result, if a cluster secret is designated to project A, an application assigned to project B will be unable to access that secret. To allow applications across various projects to use cluster secrets, you must deselect the `project` field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 PTAL at my comments and address them. Thanks!

File: modules/gitops-release-notes-1-14-0.adoc (hunk 8 above)
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-14-0.adoc:142]:
```suggestion
* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secrets applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that corresponds to the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets that are scoped to an application's project will be considered for matching a repository URL. As a result, if a cluster secret is designated for example to project A, an application assigned to project B will be unable to access that secret. To enable the use of applications across various projects for cluster secrets, you must deselect the project field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```

@anandf and @Dhruv-Soni11 what do we mean when we say unset the  `project` field here? You have used deselect here. Where is the action happening? YAML file? web console UI or the argo CD UI?

File: modules/gitops-release-notes-1-14-0.adoc (hunk 7 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:108]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc (hunk 8 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc]: Fixed.

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc]: Hi @Srivaralakshmi - I have had a detailed discussion with @anandf and due to the unavailability of a cluster, he explained the scenario on call. Basically, in this scenario, we have to remove this field from the YAML as per his information.

**Review by Srivaralakshmi — COMMENTED** [review]
/lgtm

File: modules/gitops-release-notes-1-14-0.adoc [hunk 9]
```diff
@@ -43,6 +43,30 @@ For more information, refer to the link:https://argocd-operator.readthedocs.io/e
...
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+spec:
+  server:
+    initContainers:
+    - name: <name of the initContainer> #<1>
+      image: <image name of the initContainer> #<2>
+    server:
```
Comment (karmab) [inline modules/gitops-release-notes-1-14-0.adoc:60]: isnt this line wrong?

**Comment by Preeticp** [comment]
CS ack for CM. @Dhruv-Soni11 PTAL at https://github.com/openshift/openshift-docs/pull/82209#pullrequestreview-2324069029

File: modules/gitops-release-notes-1-14-0.adoc (hunk 9 above)
Comment (iam-veeramalla) [reply modules/gitops-release-notes-1-14-0.adoc:60]: hey @Dhruv-Soni11, yeah that's a typo. You don't need `server` here.

Pasting the correct example for your reference.
```
apiVersion: argoproj.io/v1beta1
kind: ArgoCD
metadata:
  name: example-argocd
spec:
  server:
    initContainers:
    - name: <name of the initContainer> #(1)
      image: <image name of the initContainer> #(2)
    sidecarContainers:
    - name: <name of the sidecarContainer> #(3)
      image: <image name of the sidecarContainer> #(4)
```

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc]: Fixed.

**Comment by AdamSaleh** [comment]
LGTM

**Comment by kavverma** [comment]
DPM ack @Dhruv-Soni11 PTAL at https://github.com/openshift/openshift-docs/pull/82209#pullrequestreview-2324069029

**Comment by Dhruv-Soni11** [comment]
@siamaksade has provided his LGTM on a call to @Srivaralakshmi verbally. He will provide his LGTM in the PR later as he is traveling currently.

File: modules/gitops-release-notes-1-14-0.adoc [hunk 10]
```diff
@@ -81,6 +104,8 @@ The signature verification does not function with the templated project field in
 
 * Before this update, the {gitops-title} Operator included a race condition where the OpenShift Service CA created the TLS secret before the default TLS termination policy was set. As a result, the default policy was set as `Passthrough` instead of `Reencrypt`. This update fixes this race condition by verifying whether the secret was created by the OpenShift Service CA, ensuring that `Reencrypt` is used as the correct default policy. link:https://issues.redhat.com/browse/GITOPS-4947[GITOPS-4947]
 
+* Before this update, you could not log in to the Argo CD web console UI after a ROSA cluster, that includes a {gitops-shortname} Operator and an Argo CD instance configured with Dex-based SSO, resumed from hibernation. The login screen would display an error indicating an invalid redirect URI in the Dex configuration. This update fixes the issue by ensuring that the correct Dex redirect URL is updated in the Argo CD configuration whenever the Argo CD server route is modified. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```
Comment (jeana-redhat) [inline modules/gitops-release-notes-1-14-0.adoc:107]:
```suggestion
* Before this update, you could not log in to the Argo CD web console UI after a {product-rosa} cluster, that includes a {gitops-shortname} Operator and an Argo CD instance configured with Dex-based SSO, resumed from hibernation. The login screen would display an error indicating an invalid redirect URI in the Dex configuration. This update fixes the issue by ensuring that the correct Dex redirect URL is updated in the Argo CD configuration whenever the Argo CD server route is modified. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```

File: modules/gitops-release-notes-1-14-0.adoc [hunk 11]
```diff
@@ -105,4 +130,14 @@ data:
...
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secrets applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL. As a result, if a cluster secret is designated to project A, an application assigned to project B will be unable to access that secret. To allow applications across various projects to use cluster secrets, you must remove the `project` field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
+
+For more information, see link:https://argo-cd.readthedocs.io/en/latest/operator-manual/upgrading/2.11-2.12/#cluster-secret-scoping-changes[Cluster secret scoping changes].
```
Comment (jeana-redhat) [inline modules/gitops-release-notes-1-14-0.adoc:143]: - Suggest breaking this up for readability. Use the `+` between lines to make sure everything is tacked on (missing in the "For more information" line as drafted).
- Should avoid using `Applications` and `ApplicationSets` as nouns. I am not familiar with this product area, but would something like "`Application` and `ApplicationSet` resources" work?
- Also recommend moving the bug link to the end to match usual format.

```suggestion
* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value are no longer classified as global secrets applicable to all `Applications` and `ApplicationSets`. 
+
In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL. 
+
As a result, if a cluster secret is designated to project A, an application assigned to project B will be unable to access that secret. To allow applications across various projects to use cluster secrets, you must remove the `project` field. 
+
For more information, see link:https://argo-cd.readthedocs.io/en/latest/operator-manual/upgrading/2.11-2.12/#cluster-secret-scoping-changes[Cluster secret scoping changes]. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```

**Review by jeana-redhat — COMMENTED** [review]
Some format issues in the RN should be addressed (and in my opinion do not require change mgmt followup - it's just format not technical detail)

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: modules/gitops-release-notes-1-14-0.adoc [hunk 12]
```diff
@@ -43,6 +43,29 @@ For more information, refer to the link:https://argocd-operator.readthedocs.io/e
 
 * With this update, you can implement an alternative sharding algorithm that utilizes consistent hashing with bounded loads. The algorithm ensures a uniform distribution of managed clusters across shards while minimizing the need for reshuffling during configuration changes, such as adjustments in the number of managed clusters or shards. link:https://issues.redhat.com/browse/GITOPS-3584[GITOPS-3584]
 
+* With this update, {gitops-title} provides support for the `sidecar` and `init` containers for the Server and Application Controller components. You can configure the `sidecar` and `init` containers by using the `spec.server.sidecarContainers`, `spec.controller.sidecarContainers`, `spec.server.initContainers` and `spec.controller.initContainers` fields in the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-5010[GITOPS-5010]
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:46]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc [hunk 13]
```diff
@@ -43,6 +43,29 @@ For more information, refer to the link:https://argocd-operator.readthedocs.io/e
...
+metadata:
+  name: example-argocd
+spec:
+  server:
+    initContainers:
+    - name: <name of the initContainer> #<1>
+      image: <image name of the initContainer> #<2>
+    sidecarContainers:
+    - name: <name of the sidecarContainer> #<3>
+      image: <image name of the sidecarContainer> #<4>
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:62]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc (hunk 10 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:107]: Fixed.

File: modules/gitops-release-notes-1-14-0.adoc (hunk 11 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc]: Fixed.

**Comment by Dhruv-Soni11** [comment]
Hi @jeana-redhat - I have incorporated your comments in the PR and request you to merge the PR if the changes are fine.

File: modules/gitops-release-notes-1-14-0.adoc [hunk 14]
```diff
@@ -105,4 +130,18 @@ data:
...
\ No newline at end of file
+For more information, see link:https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#add-self-signed-tls-certificate-for-gitlab-scm-provider-to-applicationsets-controller[Add Self-signed TLS Certificate for Gitlab SCM Provider to ApplicationSet Controller].
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value are no longer classified as global secrets applicable to all `Applications` and `ApplicationSets`.
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-14-0.adoc:141]:
```suggestion
* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value are no longer classified as global secrets applicable to all `Applications` and `ApplicationSets` resources. 
```
Check with SME once for technical accuracy for all my suggestions. If everything checks okay, go ahead and please implement them.

File: modules/gitops-release-notes-1-14-0.adoc [hunk 15]
```diff
@@ -105,4 +130,18 @@ data:
...
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value are no longer classified as global secrets applicable to all `Applications` and `ApplicationSets`. 
++
+In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL.
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-14-0.adoc:143]:
```suggestion
In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` resource could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL. 
```

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 I have left a couple of suggestions, PTAL. Thanks!

File: modules/gitops-release-notes-1-14-0.adoc (hunk 15 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:143]: I've checked with the SME and he is in agreement with the modification suggestion. Fixed.

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-0.adoc:143]: Fixed.

**Pull request merged**